docs/examples.rst

```diff
@@ -6,8 +6,8 @@ Examples
 The pysat framework reduces the steps needed when performing certain science
 data investigations. It allows you to focus on developing routines that make
 your investigation unique, and creating the output data or visualizations you
-require. This section provides examples, with the full code for each example
-made available in the
+require. This section provides examples, with the full code for each example,
+as well as additional examples, made available in the
```

Comment:
May be worth mentioning examples created for the PyHC Summer school, https://github.com/pysat/summer-school/tree/main/pysat-tutorial

Reply:
added.